core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -222,8 +222,8 @@ public void start(Boolean isPrimary) throws IOException {
           (int) Configuration.getMs(PropertyKey.MASTER_LOG_CONFIG_REPORT_HEARTBEAT_INTERVAL)));
     } else {
       // Standby master should setup MetaMasterSync to communicate with the leader master
-      MetaMasterMasterClient metaMasterClient =
-          new MetaMasterMasterClient(MasterClientConfig.defaults());
+      RetryHandlingMetaMasterMasterClient metaMasterClient =
```

Comment:
Will the RetryHandlingMetaMasterClient make the log more clean?
The current log is a little messy, too much timeout messages.

Reply:
I renamed the class from `MetaMasterMasterClient` to `RetryHandlingMetaMasterMasterClient` without changing its functionality - do you mean the log was messy before this PR, or after?